rest/resource-server/src/test/java/org/eclipse/sw360/rest/resourceserver/restdocs/ObligationSpecTest.java

```diff
@@ -97,18 +99,31 @@ public void before() throws TException {
     public void should_document_get_obligations() throws Exception {
         String accessToken = TestHelper.getAccessToken(mockMvc, testUserId, testUserPassword);
         mockMvc.perform(get("/api/obligations")
-                .header("Authorization", "Bearer " + accessToken)
-                .accept(MediaTypes.HAL_JSON))
+                        .header("Authorization", "Bearer " + accessToken)
+                        .param("page", "0")
+                        .param("page_entries", "5")
+                        .accept(MediaTypes.HAL_JSON))
                 .andExpect(status().isOk())
                 .andDo(this.documentationHandler.document(
+                        requestParameters(
+                                parameterWithName("page").description("Page of licenses"),
```

Comment:
change description of page to page of obligations.

Reply:
Fixed